src/tpm2_wrap.c

```diff
@@ -105,8 +105,8 @@ static int wolfTPM2_Init_ex(TPM2_CTX* ctx, TPM2HalIoCb ioCb, void* userCtx,
 #else
     rc = TPM_RC_SUCCESS;
 #endif /* WOLFTPM_MCHP || WOLFTPM_PERFORM_SELFTEST */
-
-#endif /* ! WOLFTPM_LINUX_DEV */
+    (
```

Comment:
Was this `(` intentional?

Reply:
Nope, definitely an accident. Good catch. Thanks.